spec/akasha/event_router_spec.rb

```diff
@@ -7,16 +7,26 @@
     Akasha::Aggregate.connect!(repo)
   end
 
-  describe '#connect' do
-    before do
-      subject.register_event_listener(:name_changed, Notifier)
-    end
-
+  shared_examples_for 'registred actions' do
     it "routes repo's events" do
       subject.connect!(repo)
       item.name = 'new name'
       expect_any_instance_of(Notifier).to receive(:on_name_changed).with('item-1', old_name: nil, new_name: 'new name')
       repo.save_aggregate(item)
     end
   end
+
+  describe '#connect' do
+    before do
+      subject.register_event_listener(:name_changed, Notifier)
```

Comment:
I think you might drop this before and use the constructor instead as it is the preferred way to register listeners.

Reply:
Ok, I will drop it.